addons/website/tests/test_website_reset_password.py

```diff
@@ -70,13 +70,15 @@ def test_02_multi_user_login(self):
         # several users. Make sure we can still log in if 2 users exist.
         website = self.env["website"].get_current_website()
         website.ensure_one()
+        internal_group = self.env.ref('base.group_user')
+        portal_group = self.env.ref('base.group_portal')
 
         # Use AAA and ZZZ as names since res.users are ordered by 'login, name'
         self.env["res.users"].create(
-            {"website_id": False, "login": "bobo@mail.com", "name": "AAA", "password": "bobo@mail.com"}
+            {"website_id": False, "login": "bobo@mail.com", "name": "AAA", "password": "bobo@mail.com", "groups_id": [Command.link(portal_group.id), Command.unlink(internal_group.id)]}
         )
```

Comment:
This is almost doubling the line length, 185 chars, it's getting a bit too much, just split it please

Maybe just
```py
        self.env["res.users"].create({
            "website_id": False, "login": "bobo@mail.com", "name": "AAA",
            "password": "bobo@mail.com", "groups_id": [
                Command.link(portal_group.id),
                Command.unlink(internal_group.id),
            ],
        })
```